documentation/optee_design.md

```diff
@@ -315,9 +315,105 @@ TA. This is solved by always setting the TA context in the CPU when
 resuming execution. Here's room for improvements since it's more likely
 than not that it's the same CPU that resumes execution in secure world.
 
-# 6. SVC
-Will be written soon.
-
+# 6. Stacks
+Different stacks are used during different stages. The stacks are:
+- Secure monitor stack (128 bytes), bound to the CPU. Only available if
+  OP-TEE is compiled with a secure monitor always the case if the target is
+  ARMv7 but never for ARMv8.
```

Comment:
We quite often get comments from others that we should be more precise with ARMv7/v8 and write ARMv7-A, ARMv8-A instead.